packages/kbn-journeys/journey/journey_apm_config.ts

```diff
@@ -15,7 +15,7 @@ export const JOURNEY_APM_CONFIG = {
   secretToken: APM_PUBLIC_TOKEN,
   active: 'true',
   contextPropagationOnly: 'false',
-  environment: process.env.CI ? 'ci' : 'development',
+  environment: process.env.ELASTIC_APM_ENVIRONMENT || (process.env.CI ? 'ci' : 'development'),
```

Comment:
Not sure if its required, ENV variables overrides everything by default on the agent level? If this is for RUM agent, then yes its needed.

Reply:
yes, it's for the RUM agent